R/contextual_help.R

```diff
@@ -0,0 +1,129 @@
+with_help <- function(tag = NULL, help_item = NULL) {
+  # Create help button
+  help_button <- div(
+    class = "help-icon",
+    style = "
+            display: inline-block;
+            width: 16px;
+            height: 16px;
+            border-radius: 50%;
+            background-color: #007bff;
+            color: white;
+            text-align: center;
+            line-height: 16px;
+            font-size: 11px;
+            font-weight: bold;
+            margin-left: 5px;
+            cursor: pointer;
+            vertical-align: middle;
+            visibility: hidden;
```

Comment:
nemělo by to být defaultně viditelné (aby bylo jasné, že nápověda k prvku existuje)?